controllers/cloudstackaffinitygroup_controller.go

```diff
@@ -35,8 +37,9 @@ import (
 
 // CloudStackAGReconciliationRunner is a ReconciliationRunner with extensions specific to CloudStack affinity group reconciliation.
 type CloudStackAGReconciliationRunner struct {
-	csCtrlrUtils.ReconciliationRunner
+	*csCtrlrUtils.ReconciliationRunner
```

Comment:
Making this a pointer so that we can break up the base reconciler a bit at a time into implementations of interfaces that all work on the same underlying data.